www/package.json

```diff
@@ -17,6 +17,7 @@
     "gatsby-plugin-sass": "^2.0.11",
     "gatsby-source-filesystem": "^2.0.29",
     "gatsby-transformer-react-docgen": "^4.0.2",
+    "namor": "^2.0.2",
```

Comment:
What is this for?

Reply:
Lol I don't remember... I'll look at the commit and remove if it's not needed.